mlir/lib/Analysis/Presburger/Simplex.cpp

```diff
@@ -1195,4 +1195,9 @@ void Simplex::print(raw_ostream &os) const {
 
 void Simplex::dump() const { print(llvm::errs()); }
 
+bool Simplex::isRedundant(ArrayRef<int64_t> coeffs) {
+  Optional<Fraction> minimum = computeOptimum(Direction::Down, coeffs);
```

Comment:
Doesn't this assert-fail when the new inequality makes the set empty?

Reply:
I am also not 100% sure what you mean here, I guess it relates to the similar comment above.

Reply:
Nevermind, I was confused.